src/20_Components/amp-bind.html

```diff
@@ -44,10 +45,12 @@
     <script type="application/json">
       {
         "dog": {
-          "url": "/img/Border_Collie.jpg"
+          "url": "/img/Border_Collie.jpg",
+          "videoUrl": "/video/dog-video.mp4"
```

Comment:
Can we maybe find a better video? I'll try to find a better one (I know I did send you the link, but didn't watch it to the end).